app/scripts/controllers/transactions/lib/tx-state-history-helpers.js

```diff
@@ -44,9 +46,11 @@ export function generateHistoryEntry(previousState, newState, note) {
 }
 
 /**
-  Recovers previous txMeta state obj
-  @returns {Object}
-*/
+ * Recovers previous txMeta state obj
+ *
+ * @param _shortHistory
```

Comment:
In the future, is there a way to require a description or type? I know this isn't typescript, just thought I'd ask.

Reply:
Yep! That rule is part of our defaults, but is disabled here to make this diff smaller: https://github.com/MetaMask/metamask-extension/pull/12112/files#diff-e2954b558f2aa82baff0e30964490d12942e0e251c1aa56c3294de6ec67b7cf5R118

All of those disabled rules should be enabled one-by-one. Maybe taking chunks of the codebase at a time if that's easier.